php/libraries/Database.class.inc

```diff
@@ -284,6 +286,16 @@ class Database extends PEAR
     }
 
     /**
+     * Get last insert ID.
+     *
+     * @return mixed
```

Comment:
The return value isn't mixed, and it should describe what the value returned is, not just the type.